src/index.html

```diff
@@ -18,5 +23,87 @@
   </head>
   <body>
     <h1>Moyo header</h1>
```

Comment:
Use semantic tags where possible. Instead of using `<h1>` directly within the body, consider wrapping it within a `<header>` or `<section>` tag to provide a clearer structure and meaning to the document.